multisig/README.md

```diff
@@ -0,0 +1,254 @@
+## Multisig keys usage
+
+For the purpose of this doc we are considering 2 people for multisig account: Alice and Bob.
+
+### Creating individual keys and multisig address
+First lets create keys for alice on **alice's local machine**.
+```
+simd keys add alice
+
+- name: alice
+  type: local
+  address: cosmos1njefwuen6f0ava9kfp4hde03nh9va67fpspne7
+  pubkey: '{"@type":"/cosmos.crypto.secp256k1.PubKey","key":"A5U2bVyvVH5PHn2Xb9wzjMHh/utMlNTyN+OG81PCaxNJ"}'
+  mnemonic: ""
+```
+
+Then lets create keys for bob on **bob's local machine**.
+```
+simd keys add bob
+Enter keyring passphrase:
+Re-enter keyring passphrase:
+
+- name: bob
+  type: local
+  address: cosmos1se4yuxerrn77h4qc2tessfm7lnxdlrrzkj8mkf
+  pubkey: '{"@type":"/cosmos.crypto.secp256k1.PubKey","key":"Az+fBSOlALoCDmbT/3Yy8F+N24zMONLB4yuQINIAmukB"}'
+  mnemonic: ""
+
+```
+Then we need to create a multisig address which can be used by both alice and bob. But for creating a multisig address we need to have the other person's address in the keyring as well. We don't need to import the key using mnemonic, we just need the pubkey of the other account.
+
+ So lets add **bob's key** in **alice's local machine**
+ ```
+ simd keys add bob --pubkey <pubkey of bob>
+ ```
+Which will look like this in our case
+```
+simd keys add bob --pubkey '{"@type":"/cosmos.crypto.secp256k1.PubKey","key":"Az+fBSOlALoCDmbT/3Yy8F+N24zMONLB4yuQINIAmukB"}'
+```
+
+You should see an output like this
+```
+- name: bob
+  type: offline
+  address: cosmos1se4yuxerrn77h4qc2tessfm7lnxdlrrzkj8mkf
+  pubkey: '{"@type":"/cosmos.crypto.secp256k1.PubKey","key":"Az+fBSOlALoCDmbT/3Yy8F+N24zMONLB4yuQINIAmukB"}'
+  mnemonic: ""
+```
+
+Similarly we have to add **alice's key** in **bob's local machine**
+```
+simd keys add alice --pubkey '{"@type":"/cosmos.crypto.secp256k1.PubKey","key":"A5U2bVyvVH5PHn2Xb9wzjMHh/utMlNTyN+OG81PCaxNJ"}'
+Enter keyring passphrase:
+
+- name: alice
+  type: offline
+  address: cosmos1njefwuen6f0ava9kfp4hde03nh9va67fpspne7
+  pubkey: '{"@type":"/cosmos.crypto.secp256k1.PubKey","key":"A5U2bVyvVH5PHn2Xb9wzjMHh/utMlNTyN+OG81PCaxNJ"}'
+  mnemonic: ""
+
+```
+
+Now we can create a multisig address from these 2 addresses on both the local machines. Ideally this multisig address is generated on a machine which is accesible to both but we are going to have it added to both the keyrings so the need for a shared device is eliminated.
+
+We create a multisig address using the following cmd:
+```
+simd keys add multisig --multisig bob,alice --multisig-threshold 2
```

Comment:
I believe we dont' need to have the keys in our local system. We can just use pubkeys directly to add multisig